gutenberg.php

```diff
@@ -24,8 +24,15 @@
  * @since 0.1.0
  */
 function the_gutenberg_project() {
-	global $post_type_object;
-	?>
+	global $post_type_object, $post;
+
+	if ( ! wp_check_post_lock( $post->ID ) ) {
+		$active_post_lock = wp_set_post_lock( $post->ID );
+	}
+
+	if ( ! empty( $active_post_lock ) ) { ?>
+	<input type="hidden" id="active_post_lock" value="<?php echo esc_attr( implode( ':', $active_post_lock ) ); ?>" />
```

Comment:
Any reason to render this in PHP? Can't we render it in JS instead? That way we can keep it in sync using the state?

Reply:
this is essentially used to pass the locking info from php to JS which I already handle directly so i can probably drop this bit (and track the data in state as you suggested above)